_includes/content/ssh-copy-key.md

```diff
@@ -0,0 +1,73 @@
+
+<ul class="nav nav-tabs">
+  <li class="active"><a data-toggle="tab" data-target="#mac-copy-keys" data-group="mac">Mac</a></li>
+  <li><a data-toggle="tab" data-target="#win-copy-keys" data-group="win">Windows</a></li>
+  <li><a data-toggle="tab" data-target="#linux-copy-keys" data-group="linux">Linux</a></li>
+</ul>
+<div class="tab-content">
+<div id="mac-copy-keys" class="tab-pane fade in active">
+<br>
+{% capture mac-content-copy %}
+
+Copy the public SSH key to your clipboard.
+
+```none
+$ pbcopy < ~/.ssh/id_rsa.pub
+```
+
+If your SSH key file has a different name than the example code, modify the
+filename to match your current setup.
+
+>**Tip:** If you don't have `pbcopy`, you navigate to the hidden `.ssh`
+folder, open the file in a text editor, and copy it to your clipboard.
+For example: `$ atom ~/.ssh/id_rsa.pub`
+
+{% endcapture %}
+{{ mac-content-copy | markdownify }}
+</div>
+
+<div id="win-copy-keys" class="tab-pane fade">
+<br>
+{% capture win-content-copy %}
+
+Copy the public SSH key to your clipboard.
+
+```none
+$ clip < ~/.ssh/id_rsa.pub
+```
+
+If your SSH key file has a different name than the example code, modify the
+filename to match your current setup.
+
+>**Tip:** If `clip` doesn't work, navigate the hidden `.ssh`
+folder, open the file in a text editor, and copy it to your clipboard.
```

Comment:
In my experience, these admonitions work better with the `>` at the beginning of each line of the admonition. At least it helps with troubleshooting rendering problems later.